data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,408 @@ mission "Ringworld Debris: Quarg"
+			`	While you are talking with Taely, you see a young Remnant walking towards you with a touchpad and waving. After he greets you, Taely says, "Captain <first>, let me introduce you to Dusk. He is a young researcher specialized in xenoarchaeology and the study of astrobiology. Dusk, this is <first>, the captain who I told you about earlier."`
+			`	"Nice to meet you, Captain!" continues Dusk.`
+			choice
+				`	"Dusk? Are names like that common?"`
+				`	"Pleased to meet you too, Dusk."`
+					goto greet
+			`	"You mean names of the various phases of daylight? Mmm, I would not say they are common, but nor are they rare, either. Our ancestors devoted their lives, one way or another, to studying the stars and the everchanging light of the universe. It should not be surprising that their vocabulary should be so entwined in our culture."`
+			label greet
+			`	"I will let you two discuss the next steps in exploring Patir. May the embers burn bright for you both," Taely says before waking away towards the local shipyard.`
+			`	"I have found something interesting in the data Taely sent me, Captain." Dusk shows you a map of the system Patir with various lines drawn all over it. "This image shows the route followed by every Ka'sei the scanner was able to track. At first sight they do not seem to have much in common, right? However, upon closer examination, there is something that almost every path has in common: once in a while, every ship spends a few minutes on a single asteroid, one of the biggest in the entire asteroid belt.`
+			`	"However, this could have easily been just a coincidence; it takes a lot of effort to extract resources from asteroids that size, so maybe they were just combining their forces to harvest it. Unusual, but far from impossible. So, at this point I had an idea." The screen on the touchpad changes, now showing raw numbers separated on a spreadsheet. "The sensors you carried to Patir had various functions, one of which was a high power asteroid scanner. The asteroid in particular was shown to be extremely rich in a particular metal, iridium. There is more iridium in it than the whole of half a dozen Ember Waste systems, planets included. However, here's the odd part: the concentration of iridium on the asteroid when you jumped away wasn't lower than when the scanning begun, it was higher! For whatever reason, the Fetri'sei are depositing iridium on the asteroid."`
+			choice
+				`	"Are you sure that it isn't just an error in the measurement?"`
+				`	"Why are they doing that?"`
+					goto why
+			`	"Well, it is a possibility; this instrumentation hasn't been tested in years. But if it is not, this could give us a lot of insight on the real purpose of these Ka'sei and why they are around that black hole. It is at least worth studying!`
```

Comment:
```suggestion
			`	"Well, it is a possibility; this instrumentation has not been tested in years. But if it is not, this could give us a lot of insight on the real purpose of these Ka'sei and why they are around that black hole. It is at least worth studying!`
```